homework-submission/week1-react/src/App.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import todoLogo from './todo.png';
+import './App.css';
+import {inject, observer} from 'mobx-react';
+import Todo from "./component/todo.js";
+import Add from "./component/add";
+import Update from "./component/update";
+
+@inject('myTodoApp')
+@observer
+
+class App extends Component {
+
+  render() {
+    const {myTodoApp} = this.props;
+
+    const isEmpty = myTodoApp.data.todos.length < 1 || myTodoApp.data.todos === undefined;
+
+    const todoList = myTodoApp.data.todos.map((item, key)=> 
+    (key === myTodoApp.data.indexToUpdate) ?
+    <Update
```

Comment:
This is returning a `form` which is not a valid child for `ul` element.

Reply:
Yes, I made change and now it is like below:
```
<li>
      <form> 
                  ...
      <form/>
<li/>
```